### PR TITLE
Show service versions

### PR DESCRIPTION
**Pull request by nazywam**
~~**This requires https://github.com/CERT-Polska/karton/pull/93 to be merged and released and the karton-core version in requirements to be bumped**~~

This PR introduces a new badge for the `version` field exposed in consumers.

image

### CODE REVIEW

File: karton/dashboard/templates/index.html [hunk 1]
```diff
@@ -19,7 +19,14 @@ <h3 class="text-center">binds</h3>
         <td>
           <a href="queue/{{ queue_name }}">{{ queue_name }}</a>
           <div>
-            <span class="badge badge-info">{{ queue.bind.version }}</span>
+            <span class="badge badge-info">
+              📦{{ queue.bind.version }}
+            </span>
```
Comment (msm-code) [inline karton/dashboard/templates/index.html:24]: Maybe add onmouseover tooltips to both of them? To make it more obvious what these versions are and how to interpret this.

Just "karton-core library version" and "version of installed package for this service" (or similar) should be enough I think.

File: karton/dashboard/templates/index.html [hunk 2]
```diff
@@ -19,7 +19,14 @@ <h3 class="text-center">binds</h3>
         <td>
           <a href="queue/{{ queue_name }}">{{ queue_name }}</a>
           <div>
-            <span class="badge badge-info">{{ queue.bind.version }}</span>
+            <span class="badge badge-info">
+              📦{{ queue.bind.version }}
```
Comment (msm-code) [inline karton/dashboard/templates/index.html:23]: no `v` here but there is one below. Looks inconsistent :thinking:

File: karton/dashboard/templates/index.html (hunk 1 above)
Comment (nazywam) [reply karton/dashboard/templates/index.html:24]: image

`title` seems to do the job

File: karton/dashboard/templates/index.html (hunk 2 above)
Comment (nazywam) [reply karton/dashboard/templates/index.html:23]: I've thought that we either use :package: or `v`, but it sill looks okay :+1:

File: karton/dashboard/templates/queue_layout.html [hunk 3]
```diff
@@ -29,10 +29,14 @@ <h3 class="text-center">queue {{ name }}</h3>
           </div>
       {% endfor %}
     </dd>
-    <dt class="col-3">Library version</dt>
+    <dt class="col-3">Karton library version</dt>
```
Comment (nazywam) [inline karton/dashboard/templates/queue_layout.html:32]:
```suggestion
    <dt class="col-3">Karton-core library version</dt>
```

**Review by msm-code — APPROVED** [review]
:+1:

**Review by psrok1 — APPROVED** [review]

**Pull request merged**